products/mssqlserver.md

```diff
@@ -11,135 +11,408 @@ versionCommand: select @@version
     codename: Denali
     releaseLabel: "2012 SP4"
     releaseDate: 2017-10-05
-    support: 2017-07-11
+    support: 2022-07-12
     eol: 2022-07-12
     extendedSupport: 2025-07-08
-    latest: "11.0.7512.11"
+    latestGdr: "11.0.7512.11"
+    latestGdrLink: https://support.microsoft.com/help/5021123
+    latest: "11.0.7512.11 GDR"
     latestReleaseDate: 2023-02-14
     link: https://support.microsoft.com/help/5021123
 
--   releaseCycle: "2008 R2"
+-   releaseCycle: "14.0"
+    codename: Helsinki
+    releaseLabel: "2017"
+    releaseDate: 2017-09-29
+    support: 2022-10-11
+    eol: 2027-10-12
+    extendedSupport: false
+    latestGdr: "14.0.2052.1"
+    latestGdrLink: https://support.microsoft.com/help/5029375
+    latest: "14.0.3465.1 CU31+GDR"
+    latestReleaseDate: 2023-10-10
+    link: https://support.microsoft.com/help/5029376
+
+-   releaseCycle: "13.0-sp1"
+    codename: SQL16
+    releaseLabel: "2016 SP1"
+    releaseDate: 2016-11-16
+    support: 2019-07-09
+    eol: 2019-07-09
+    extendedSupport: false
+    latestGdr: "13.0.4259.0"
+    latestGdrLink: https://support.microsoft.com/help/4505219
+    latest: "13.0.4604.0 CU15+GDR"
+    latestReleaseDate: 2019-07-09
+    link: https://support.microsoft.com/help/4505221
+
+-   releaseCycle: "12.0-sp2"
+    codename: Hekaton
+    releaseLabel: "2014 SP2"
+    releaseDate: 2016-07-14
+    support: 2020-01-14
+    eol: 2020-01-14
+    extendedSupport: false
+    latestGdr: "12.0.5223.6"
+    latestGdrLink: https://support.microsoft.com/help/4505217
+    latest: "12.0.5687.1 CU18"
+    latestReleaseDate: 2019-07-29
+    link: https://support.microsoft.com/help/4500180
+
+-   releaseCycle: "13.0"
+    codename: SQL16
+    releaseLabel: "2016"
+    releaseDate: 2016-06-01
+    support: 2018-01-09
+    eol: 2018-01-09
+    extendedSupport: false
+    latestGdr: "13.0.1745.2"
+    latestGdrLink: https://support.microsoft.com/help/4058560
+    latest: "13.0.2218.0 CU9+GDR"
+    latestReleaseDate: 2018-01-06
+    link: https://support.microsoft.com/help/4058559
+
+-   releaseCycle: "11.0-sp3"
+    codename: Denali
+    releaseLabel: "2012 SP3"
+    releaseDate: 2015-12-01
+    support: 2018-10-09
+    eol: 2018-10-09
+    extendedSupport: false
+    latestGdr: "11.0.6260.1"
+    latestGdrLink: https://support.microsoft.com/help/4057115
+    latest: "11.0.6614.2 CU10+QFE"
+    latestReleaseDate: 2018-01-16
+    link: https://support.microsoft.com/help/4057121
+
+-   releaseCycle: "12.0-sp1"
+    codename: Hekaton
+    releaseLabel: "2014 SP1"
+    releaseDate: 2015-04-14
+    support: 2017-10-10
+    eol: 2017-10-10
+    extendedSupport: false
+    latestGdr: "12.0.4237.0"
+    latestGdrLink: https://support.microsoft.com/help/4019091
+    latest: "12.0.4522.0 CU13"
+    latestReleaseDate: 2017-08-08
+    link: https://support.microsoft.com/help/4019099
+
+-   releaseCycle: "10.50-sp3"
     codename: Kilimanjaro
     releaseLabel: "2008 R2 SP3"
     releaseDate: 2014-09-26
```

Comment:
```suggestion
    releaseDate: 2014-07-07
```

According to https://learn.microsoft.com/en-us/lifecycle/products/microsoft-sql-server-2008-r2.